### ISSUE TITLE
sf2d deprecated

### ISSUE THREAD

**Issue by IvanFon** [BtheDestroyer/SpriteTools#8]
[sf2dlib has been deprecated](https://github.com/xerpi/sf2dlib/blob/8da504f2f3bd024474e65d91d8173fe111d6ece3/README.md). What does that mean for this project? Will it be removed and replaced with just citro3d?

**Comment by Bryce-Dixon** [comment]
Yes, that's exactly correct. I'm currently finalizing my plans for the summer, so once that's done I'll be working on that as well as learning/adding OpenGL to the project as a front-end for testing purposes.

Thank you for your concern and continued support of the project, everyone.

**Comment by GabrielRRussell** [comment]
Isn't Citro3D Cpp only? That's gonna be one hell of a re-write.

**Comment by Bryce-Dixon** [comment]
If Citro3D was C++ only then SF2D wouldn't have been possible in standard C as it used Citro3D. Besides, you can compile Cpp to function with standard C code.
